frontend/components/containers/annotation/ApproveButton.vue

```diff
@@ -2,11 +2,13 @@
   <v-tooltip bottom>
     <template v-slot:activator="{ on }">
       <v-btn
+        v-shortkey.once="['a']"
```

Comment:
The short key `a` is not appropriate if some label has the same short key.

![](https://gyazo.com/dbcb1074c347a841a274d8aa58027a2a/raw)

Reply:
Fair point. Do you have a preference on what a more appropriate short key would be?

Reply:
I like the `enter` key.

The idea of using two keys is a good idea. But actually, I implemented the database scheme to be able to use a combination of two/three keys(ctrl, shift, ctrl+shift, and [0-9][a-z]) as a short key of a label. So we need to avoid using shortcut keys in that range.

Reply:
Makes sense to me. I just committed a change to use ```enter``` instead of ```a```.